### ISSUE TITLE
1.18.1 Crash before opening

### ISSUE THREAD

**Issue by Baggytrousers27** [Insane96/ProgressiveBosses#114]
Crash back to launcher immediately after launching

**Versions:**
 - Minecraft: 1.18.1
 - Forge: 39.0.5
 - Progressive Bosses: ProgressiveBosses-Alpha-3.5.0-mc1.18.1
 - InsaneLib: InsaneLib-1.4.2-mc1.18.1

[Game output log](https://pastebin.com/jLQ2Q08d)

**Comment by ghost** [comment]
The specific error, although I don't have the log available rn, is something about the version number ('Alpha-3.5.0') not starting with a number.

**Comment by Insane96** [comment]
If you could send more logs (e.g. crash logs) would be really useful.

**Comment by ghost** [comment]
Sure, of course, I have it now. There's no crash log, because minecraft fails to start, but here are the last lines of the console:

``` <log4j:Event logger="net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer" timestamp="1640098272335" level="INFO" thread="main">
    <log4j:Message><![CDATA[Found mod file forge-1.18.1-39.0.6-universal.jar of type MOD with locator net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@18f20260]]></log4j:Message>
  </log4j:Event>

instrument.ts:130 2021-12-21 08:51:12,482 main WARN Error parsing URI ..\..\datastore\assets\objects\bd\client-1.12.xml

instrument.ts:130 ps stderr: Exception in thread "main" 
(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: java.lang.IllegalArgumentException: Alpha-3.5.0: Version string does not start with a number

(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: 	at java.base/java.lang.module.ModuleDescriptor$Version.<init>(ModuleDescriptor.java:1005)
	at java.base/java.lang.module.ModuleDescriptor$Version.parse(ModuleDescriptor.java:1090)

(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: 	at java.base/java.lang.module.ModuleDescriptor$Builder.version(ModuleDescriptor.java:2062)
	at MC-BOOTSTRAP/fmlloader@1.18.1-39.0.6/net.minecraftforge.fml.loading.moddiscovery.ModJarMetadata.descriptor(ModJarMetadata.java:67)
	at cpw.mods.securejarhandler@0.9.54/cpw.mods.jarhandling.impl.Jar.computeDescriptor(Jar.java:47)
	at cpw.mods.securejarhandler@0.9.54/cpw.mods.cl.JarModuleFinder$JarModuleReference.<init>(JarModuleFinder.java:50)
	at cpw.mods.securejarhandler@0.9.54/cpw.mods.cl.JarModuleFinder.lambda$new$0(JarModuleFinder.java:28)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)

(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: 	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at cpw.mods.securejarhandler@0.9.54/cpw.mods.cl.JarModuleFinder.<init>(JarModuleFinder.java:29)
	at cpw.mods.securejarhandler@0.9.54/cpw.mods.cl.JarModuleFinder.of(JarModuleFinder.java:43)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:61)

(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)

(anonymous) @ instrument.ts:130
instrument.ts:130 ps stderr: 	at cpw.mods.bootstraplauncher@0.1.17/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:90)

(anonymous) @ instrument.ts:130
instrument.ts:130 Process exited with code 1. Not too good..```

**Comment by Insane96** [comment]
Thanks. Re-released 3.5.0 without the "alpha" before the version number.
